Refactor the branding documentation (#73)
owncloud/docs-client-branding: modules/ROOT/pages/index.adoc

= Branded Clients
:toc: right

> :description: Welcome to the ownCloud Branded Apps documentation. This guide provides information on how to brand, update, deploy or distribute your branded client.

Author: ```suggestion
:description: Welcome to the ownCloud Branded Clients documentation. This guide provides information on how to brand, update, deploy or distribute your branded client.
```

== Introduction

{description}

> When you are interested in branding your client, you need to have a branding subscription. Please use the {oc-contact-url}[ownCloud contact form] to get in touch.

Reviewer: ```suggestion
If you are interested in branding your client, you need to have a branding subscription. Please use the {oc-contact-url}[ownCloud contact form] to get in touch.
```

> When you are interested in branding your client, you need to have a branding subscription. Please use the {oc-contact-url}[ownCloud contact form] to get in touch.

Reviewer 2: ```suggestion
When you are interested in branding your client, you need to have a subscription. Please use the {oc-contact-url}[ownCloud contact form] to get in touch.
```

Self-service subscription also included in enterprise subscription.

== Branding Benefits

> Compared to standard clients public available, you can provide your customers and/or users a branded client with whitelabeling, theming, feature customization etc. This can be very beneficial to have the corporate look and feel, predefined or disabled settings, profiles and update services just to name some. 

Reviewer 3: ```suggestion
Compared to standard clients publicly available, you can provide your customers and/or users a branded client with white-labeling, theming, feature customization etc. This can be very beneficial to have the corporate look and feel, predefined or disabled settings, profiles and update services just to name some. 
```

> Compared to standard clients publicly available, you can provide your customers and/or users a branded client with white-labeling, theming, feature customization etc. This can be very beneficial to have the corporate look and feel, predefined or disabled settings, profiles and update services just to name some. 

Reviewer: ```suggestion
Compared to standard clients publicly available, you can provide your customers and/or users with a branded client with white-labeling, theming, feature customization etc. It can be very beneficial to have the corporate look and feel, predefined or disabled settings, profiles and update services just to name some. 
```

== Target Audience

> This documetation focues on the processes and how-to and is therefore addressed to technical oriented staff.

Reviewer 3: ```suggestion
This documentation focuses on the processes and how-to and is therefore addressed to technically-oriented staff.
```
